python/llm/example/CPU/Application/hf-agent/README.md

```diff
@@ -0,0 +1,81 @@
+# BigDL-LLM Transformers INT4 Optimization for HuggingFace Transformers Agent
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on HuggingFace Transformers Agent. For illustration purposes, we utilize the [lmsys/vicuna-7b-v1.5](https://huggingface.co/lmsys/vicuna-7b-v1.5) as the reference model.
+
+## 0. Requirements
```

Comment:
Remove Requirements and Best Known Configurations, and put a reference to https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model#recommended-requirements as other examples in HF-Transformers-AutoModels/Model